diff --git a/plotjuggler_base/include/PlotJuggler/special_messages.h b/plotjuggler_base/include/PlotJuggler/special_messages.h
index d025dcb0..c80f7352 100644
--- a/plotjuggler_base/include/PlotJuggler/special_messages.h
+++ b/plotjuggler_base/include/PlotJuggler/special_messages.h
@@ -6,6 +6,7 @@
 
 #include <vector>
 #include <string>
+#include <cstdint>
 
 namespace PJ::Msg
 {
diff --git a/plotjuggler_plugins/DataLoadULog/ulog_parser.h b/plotjuggler_plugins/DataLoadULog/ulog_parser.h
index 8fa0e370..c7ce3481 100644
--- a/plotjuggler_plugins/DataLoadULog/ulog_parser.h
+++ b/plotjuggler_plugins/DataLoadULog/ulog_parser.h
@@ -5,6 +5,7 @@
 #include <map>
 #include <set>
 #include <string.h>
+#include <cstdint>
 
 #include "string_view.hpp"
 
